lib/services/task_instance_service.dart

```diff
@@ -17,9 +18,9 @@ class TaskInstanceService {
 			var task = await _dataRepository.getTask(taskId: taskInstances[i].taskID);
 			if(taskUiTypes[i] == TaskUIType.currentlyPerformed || taskUiTypes[i] == TaskUIType.inBreak) {
```

Comment:
That should be a little more elegant (up to 23 line ofc but github is retarded)
```suggestion
			int Function() elapsedTimePassed;
			if(taskUiTypes[i] == TaskUIType.currentlyPerformed || taskUiTypes[i] == TaskUIType.inBreak)
				elapsedTimePassed = () => taskUiTypes[i] == TaskUIType.currentlyPerformed ? sumDurations(taskInstances[i].duration).inSeconds : sumDurations(taskInstances[i].breaks).inSeconds;
			uiTaskInstances.add(UITaskInstance.listFromDBModel(task: taskInstances[i], name: task.name, description: task.description, points: task.points != null ? UIPoints(quantity: task.points.quantity, type: task.points.icon, title: task.points.name) : null, type: taskUiTypes[i], elapsedTimePassed: elapsedTimePassed));
```